docs/cloud-workload-security/linux_expressions.md

```diff
@@ -38,6 +38,7 @@ Triggers are events that correspond to types of activity seen by the system. The
 | `chown` | File | A file’s owner was changed | 7.27 |
 | `connect` | Network | A connect was executed | 7.60 |
 | `dns` | Network | A DNS request was sent | 7.36 |
+| `dns_response` | Network | A DNS response was received | 7.6x |
```

Comment:
could we use `dns.response` instead of dns_response ? to have a more consistent syntax. In case of a response both dns.question and dns.response will be filled :thinking: